examples/foundationCentral/api_keys/main.tf

```
@@ -0,0 +1,41 @@
+// resources/datasources used in this file were introduced in nutanix/nutanix version 1.5.0-beta.2
+terraform {
+    required_providers {
+      nutanix = {
+          source = "nutanix/nutanix"
+          version = ">1.5.0-beta.2"
```

```suggestion
          version = ">=1.5.0-beta.2"
```
 >= as we are releasing this in 1.5.0-beta.2 right ?